### DISCUSSION TITLE
Bug in quizsliver

### DISCUSSION THREAD

**Discussion by abdulsattar0x01** [nmanumr/quizsliver#5] [General]
![image](https://user-images.githubusercontent.com/73302708/135816148-fd975584-6c8e-4d17-995e-96384eeb8f15.png)
having this bug can you fix it?

**Comment by nmanumr** [comment]
The installation instructions have been changed please make sure you are following them correctly: https://github.com/nmanumr/quizsliver#how-to-install

Also, look like you are not select the right path of the extension. or maybe you extracted everything on the desktop folder which is bad.

**Reply by muhammadziyad009** [reply]
hey man we need and updated version as 1.3.5 is not working on cui sahiwal
